integrated_channels/canvas/utils.py

```diff
@@ -171,3 +171,33 @@ def find_course_by_course_id(
                     'Found course under root Canvas account'
                 ))
         return course
+
+    @staticmethod
+    def course_create_endpoint(enterprise_configuration):
+        """
+        Returns endpoint to POST to for course creation
+        """
+        return '{}/api/v1/accounts/{}/courses'.format(
+            enterprise_configuration.canvas_base_url,
+            enterprise_configuration.canvas_account_id,
+        )
+
+    @staticmethod
+    def course_update_endpoint(enterprise_configuration, course_id):
+        """
+        Returns endpoint to PUT to for course update
+        """
+        return '{}/api/v1/courses/{}'.format(
+            enterprise_configuration.canvas_base_url,
+            course_id,
+        )
+
+    @staticmethod
```

Comment:
great at some point whoever of us touches the code next should probably colocate all the urls in the CanvasUtil, for easy ref of what urls we use, for one thing.